xlators/features/quota/src/quotad-helpers.c

```diff
@@ -36,7 +36,7 @@ get_quotad_aggregator_state(xlator_t *this, rpcsvc_request_t *req)
 
     state->itable = active_subvol->itable;
 
-    state->pool = this->ctx->pool;
+    state->pool = global_ctx->pool;
```

Comment:
Do we need to save the global pool in a private structure ? we could use the global one where required.